src/switchroot/ostree-remount.c

```diff
@@ -92,6 +100,7 @@ main(int argc, char *argv[])
           if (errno != EINVAL)
             err (EXIT_FAILURE, "failed to remount %s", target);
         }
+      printf ("Remounted: %s\n", target);
```

Comment:
But it'll still print it if we get `EINVAL`, right? That can be misleading. (Heck, maybe we should even print a different msg in the `EINVAL` case).